raiden/routing.py

```diff
@@ -36,7 +36,20 @@ def get_best_routes(
 ) -> List[RouteState]:
     services_config = config.get("services", None)
 
-    if services_config and services_config["pathfinding_service_address"] is not None:
+    if to_address in views.all_neighbour_nodes(chain_state):
+        neighbours = get_best_routes_internal(
+            chain_state=chain_state,
+            token_network_id=token_network_id,
+            from_address=from_address,
+            to_address=to_address,
+            amount=amount,
+            previous_address=previous_address,
+        )
+        for route_state in neighbours:
+            if to_address == route_state.node_address:
+                return [route_state]
```

Comment:
@rakanalh @hackaugusto When doing a transfer we want to use the direct channel if possible. However, the capacity of the channel will be checked later, so this will not work when the channel doesn't have sufficient capacity. Is it OK to check the capacity before?

Reply:
The reason we did not check capacity in the first place was the fact that we wanted to have (at some point) the transfer wait until both enough capacity / node availability conditions are met for the mediators to forward the transfer, as capacity/availability might change during the transfer's lifecycle. We've implemented certain state changes to notify about availability but not capacity when we removed the checks here in the routing module but not capacity change notifications. For now, i guess there's no harm in restoring checks for capacity until we decide to remove it.

Reply:
Should we introduce the capacity check for all the nodes again or just for the direct transfer?